aspnet-core/GettingStarted/Getting-Started-2-0.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
 title: Getting Started | ASP.NET Core | Syncfusion
-description: Getting Started.
+description: Learn here more about getting started with Syncfusion Essential ASP.NET Core Common Control, its elements, and more.
```

Comment:
Not common control, check and change in description and h1 tag

Reply:
Done